### PR TITLE
[map] zoom in on map

### PR DESCRIPTION
**Pull request by chejennifer**
-added ability to zoom and pan. To zoom in and out, can use buttons or mouse wheel

dev updated (http://dev.datacommons.org/tools/map#%26sv%3DCumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased%26pc%3D1)

screen-recording (17)

### CODE REVIEW

File: static/js/chart/draw_choropleth.ts [hunk 1]
```diff
@@ -81,7 +81,9 @@ function drawChoropleth(
   canClick: boolean,
   getRedirectLink: (geoDcid: GeoJsonFeatureProperties) => string,
   getTooltipHtml: (place: NamedPlace) => string,
-  zoomDcid?: string
+  zoomDcid?: string,
```
Comment (beets) [inline static/js/chart/draw_choropleth.ts:84]: please document this growing list of parameters :)

**Review by beets — APPROVED** [review]
very very cool!

i think the changes aren't on dev anymore, but looking at the video - would be nice if hover is disabled during the zoom transitions.

File: static/js/chart/draw_choropleth.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/chart/draw_choropleth.ts:102]: done

**Comment by chejennifer** [comment]
> very very cool!
> 
> i think the changes aren't on dev anymore, but looking at the video - would be nice if hover is disabled during the zoom transitions.

Thanks! I will try to disable hover during zoom transitions in a separate pr

**Pull request merged**